tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-cpu.yml

```diff
@@ -0,0 +1,102 @@
+trigger: none
+
+stages:
+- stage: Python_Packaging
+
+  jobs:
+    - job: Linux_Training_CPU_Wheels
+      timeoutInMinutes: 60
+      workspace:
+        clean: all
+      pool: Onnxruntime-Linux-GPU-NV6
+
+      variables:
+        PythonVersion: $(PYTHON_VERSION)
+        DockerFile: 'Dockerfile.manylinux2014_cpu'
+
+      steps:
+      - checkout: self
+        clean: true
+        submodules: recursive
+
+      - template: templates/set-python-manylinux-variables-step.yml
+
+      - template: templates/get-docker-image-steps.yml
+        parameters:
+          Dockerfile: tools/ci_build/github/linux/docker/$(DockerFile)
+          Context: tools/ci_build/github/linux/docker
+          DockerBuildArgs: >-
+            --build-arg PYTHON_VERSION=$(PythonVersion)
+            --build-arg INSTALL_DEPS_EXTRA_ARGS=-tu
+            --build-arg BUILD_UID=$(id -u)
+          Repository: onnxruntimetrainingcpubuild
+
+      - task: CmdLine@2
+        displayName: 'build onnxruntime'
+        inputs:
+          script: |
+            mkdir -p $HOME/.onnx
+            docker run --rm -e CC=/opt/rh/devtoolset-8/root/usr/bin/cc -e CXX=/opt/rh/devtoolset-8/root/usr/bin/c++ -e CFLAGS="-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -fcf-protection -O3 -Wl,--strip-all" -e CXXFLAGS="-Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -fstack-clash-protection -fcf-protection -O3 -Wl,--strip-all" \
+              --volume /data/onnx:/data/onnx:ro \
+              --volume $(Build.SourcesDirectory):/onnxruntime_src \
+              --volume $(Build.BinariesDirectory):/build \
+              --volume /data/models:/build/models:ro \
+              --volume $HOME/.onnx:/home/onnxruntimedev/.onnx \
+              -e NIGHTLY_BUILD \
+              -e BUILD_BUILDNUMBER \
+              onnxruntimetrainingcpubuild \
+                $(PythonManylinuxDir)/bin/python3 /onnxruntime_src/tools/ci_build/build.py \
+                  --build_dir /build \
+                  --config Release \
+                  --skip_submodule_sync \
+                  --parallel \
+                  --build_wheel \
+                  --enable_training \
+                  --update \
+                  --build \
+                  --cmake_extra_defines CMAKE_CUDA_HOST_COMPILER=/opt/rh/devtoolset-8/root/usr/bin/cc PYTHON_INCLUDE_DIR=/opt/python/cp36-cp36m/include/python3.6m PYTHON_LIBRARY=/usr/lib64/librt.so ;
```

Comment:
Do you need to set CMAKE_CUDA_HOST_COMPILER if it is cpu only?

Reply:
removed the option